front_end/audits2/Audits2Panel.js

```diff
@@ -1,43 +1,242 @@
 // Copyright (c) 2016 The Chromium Authors. All rights reserved.
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
+
+/**
+ * @typedef {{
+ *     lighthouseVersion: ?string,
+ *     generatedTime: !string,
+ *     initialUrl: !string,
+ *     url: !string,
+ *     audits: ?Object,
+ *     aggregations: !Array.<*>
+ * }}
+ */
+Audits2.LighthouseResult;
+
+/**
+ * @typedef {{
+ *     bloburl: !string,
+ *     lighthouseResult: ?Audits2.LighthouseResult
```

Comment:
will this ever be null? Seems like it should be `lighthouseResult: !Audits2.LighthouseResult` based on `start()` where this comes from